community/csv/src/main/java/org/neo4j/csv/reader/Extractors.java

```diff
@@ -1028,9 +1040,11 @@ public AnyValue value()
         {
             super( "Date" );
         }
+
         @Override
-        protected boolean extract0( char[] data, int offset, int length )
+        protected boolean extract0( char[] data, int offset, int length, AnyValue[] optionalData )
         {
+            //TODO: Change DateValue parse to use optionalData
```

Comment:
So this will happen in a different PR I guess?

Reply:
Correct!